src/lammpsio/data.py

```diff
@@ -222,6 +309,15 @@ def read(self):
             box_bounds = [None, None, None, None, None, None]
             box_tilt = None
             num_types = None
+            N_bonds = None
```

Comment:
Can we add `num_types` as a property of the `Snapshot` for the particles, like you did for `Topology`?